src/index.js

```diff
@@ -17,6 +17,14 @@ const router = new koaRouter()
 createAPIRoutes(router)
 createMiddlewareRoute(router)
 
+router.get('/_health', (ctx, next) => {
+  ctx.status = 200
+  ctx.body = {
+    status: 'UP'
```

Comment:
Maybe we should look at updating this value to be a timestamp of the running node process. This will give us an indication of how long it has been up for

Reply:
Sounds good :+1: